src/components/Form/Form.js

```diff
@@ -41,7 +41,7 @@ function Form() {
 const FormDiv = styled.div`
 padding: 0.5rem 0;
 `
-const FormSubmit = styled.input`
+const FormSubmit = styled.button`
```

Comment:
Please rename it: 
```suggestion
const SubmitButton = styled.button`
```
And implement it like this:
```
<SubmitButton>Submit</SubmitButton>
```